packages/modules/viewer-react/src/tests/components/iModel/IModelLoader.test.tsx

```diff
@@ -126,6 +126,8 @@ jest.mock("@itwin/core-frontend", () => {
     SheetViewState: {
       className: "",
     },
+    FrontendLoggerCategory: {},
```

Comment:
These mocks are getting out of control and make our tests feeble. While I know this Viewer is on the back burner, it does seem like it's still being used as an initial testing ground for new features for our primary viewer. We may want to take a look at this.

Reply:
We could look into switching to vitest?

Reply:
I don't think the issue is with the library - it's how we use it. You can mock the world with any test library, that doesn't mean it's a good idea.

Reply:
But yeah I like vitest :)